proptest/src/bits.rs

```diff
@@ -451,6 +451,7 @@ pub(crate) mod varsize {
     pub(crate) struct VarBitSet(Inner);
 
     impl VarBitSet {
+        #[allow(dead_code)] // todo: cleanup redundant code.
```

Comment:
Maybe add a tracking issue for clean-up?

Reply:
#2004

Reply:
32ce7073475692638acfc594ebb94aa94d82c1d5